gcp-js-webserver/index.js

```diff
@@ -1,10 +1,20 @@
 const gcp = require("@pulumi/gcp");
 
-const computeNetwork = new gcp.compute.Network("network", {
-    autoCreateSubnetworks: true,
+const computeNetwork = new gcp.compute.Network("webserver", {
+    autoCreateSubnetworks: false,
 });
 
-const computeFirewall = new gcp.compute.Firewall("firewall", {
+const subnetwork = new gcp.compute.Subnetwork("webserver", {
+    network: computeNetwork.selfLink,
+    ipCidrRange: "10.2.0.0/16",
+    region: "us-central1",
```

Comment:
Do we hard code us-central1 for this?  I would have assumed this comes from config?

Reply:
Switched to using config.